hacspec-derive/tests/test_derive.rs

```diff
@@ -0,0 +1,56 @@
+use hacspec::prelude::*;
+use hacspec_derive::*;
+
+#[allow(dead_code)]
+#[derive(Default, Clone, Debug, Numeric)]
```

Comment:
From these I guess we always want to have at least `Default`, probably also `Debug`. It would be great if we could pull as much inside the `Numeric` as possible.

Reply:
See previous comment